contrib/sssd.spec.in

```diff
@@ -1,7 +1,7 @@
 # SSSD SPEC file for Fedora 34+ and RHEL-9+
 
 # define SSSD user
-%if 0%{?rhel}
+%if 0%{?fedora} >= 41 || 0%{?rhel}
```

Comment:
Hi,

wouldn't it be more convenient to have this `%if` only once and e.g. set `%global use_sssd_user` which can be checked in the other places? This would it make more easy to understand which other setting might be important for running as SSSD user.

bye,
Sumit

Reply:
Good idea. Done.